Lib9c/Model/Item/Costume.cs

```diff
@@ -8,12 +8,13 @@
 namespace Nekoyume.Model.Item
 {
     [Serializable]
-    public class Costume : ItemBase, INonFungibleItem
+    public class Costume : ItemBase, INonFungibleItem, IEquippableItem
     {
         public bool equipped = false;
```

Comment:
this field can be private

Reply:
@ipdae We already use this field in `AvatarState`, `Sell2`, `Sell3`..😭

Reply:
😅 Could you please add a comment this field not to use anymore?

Reply:
And.. I find some week logic https://github.com/boscohyun/lib9c/blob/e3ea312f2f806672db7dc7fef5d112bb98e8d3c4/Lib9c/Model/State/AvatarState.cs#L522